.github/workflows/release-discord.yml

```diff
@@ -1,21 +1,37 @@
 name: Notify Discord on Release
 
 on:
-    push:
-      tags:
-        - '*'
-  
-    # Allows you to run this workflow manually from the Actions tab
-    workflow_dispatch:
+  releases:
+    types: [published]
 
 jobs:
-  # See https://github.com/teneplaysofficial/releasify-action
-  # and https://dev.to/tene/introducing-releasify-action-announce-github-releases-on-discord-effortlessly-3132
-  discordNotification:
+  notify:
     runs-on: ubuntu-latest
     steps:
-      - name: Notify via Releasify
-        uses: teneplaysofficial/releasify-action@v1
```

Comment:
releasify-action didn't format the release message correctly (screenshot below), so instead I'm replacing it with manually constructing the message.

<img width="610" height="321" alt="Screenshot 2025-10-05 at 09 20 37" src="https://github.com/user-attachments/assets/42381ad5-e671-43dc-a764-963d56ce266f" />